docs/introduction/record_agent.md

```diff
@@ -6,25 +6,30 @@ title: Recording Agents
 # Recording Agents
 
 ```{eval-rst}
- .. py:currentmodule: gymnasium.wrappers
+.. py:currentmodule: gymnasium.wrappers
 
-During and after training an agnt, it is interesting to record how the agent acts over an episode, in particular to video the agent behaviour and log the total reward for each episode. This can be achieved through two wrappers: :class:`RecordEpisodeStatistics` and :class:`RecordVideo`, the first will track episode data such as the total rewards, episode length and time taken with the second saving the environment rendering as mp4 videos.
+During training or evaluating an agent, developers often record how the agent acts over an episode, in particular, video the agent behaviour and log the total reward for each episode. Gymnasium simplifies this process with two wrappers: :class:`RecordEpisodeStatistics` and :class:`RecordVideo`, the first tracks episode data such as the total rewards, episode length and time taken and the second recording environment renderings as mp4 videos.
```

Comment:
During training or when* evaluating an agent

... and the second records* environment renderings